### ISSUE TITLE
Fixnum to String

### ISSUE THREAD

**Issue by ohnnyj** [vigetlabs/sass-json-vars#19]
First off thanks for this.

I am running into an issue, namely:

"Can't convert Fixnum to String"

Which can be traced to importer.rb:

```ruby
def _make_map(item)
  '(' + item.map {|key, value| key.to_s + ':' + _convert_to_sass(value) }.join(',') + ')'
end
```

Problem seems to only occur when you have a nested json object with a numeric value, i.e.:

```json
{
  "meh": {
    "hello": 100
  }
}
```

If I change _convert_to_sass(value) to _convert_to_sass(value).to_s everything works as expected.

**Comment by Teepo** [comment]
Hi.

In fact I think that the parser only handles Strings:

```javascript
{ 
    "data": {
        "value": true
     }
}
```

Return an error: `TypeError: can not convert true Into String`

**Comment by ensk** [comment]
hi 

i have the same problem and only occurs with json objects... any idea to solve this ?

**Comment by nhunzaker** [comment]
Hey everyone. Sorry for the delay here...

Looks like there was an error in the Ruby parser. I've addressed it in this PR:
https://github.com/vigetlabs/sass-json-vars/commit/a42ed618cdc1ae81df95a32dd099f26816989d1e

Hopefully we can get this out shortly

**Comment by nhunzaker** [comment]
Fixed in `0.3.3`!
